.travis.yml

```diff
@@ -18,3 +18,9 @@ deploy:
     node: '4'
     tags: true
     repo: thelounge/lounge
+
+script:
```

Comment:
They would fail at `npm install` time. I think this is good as is for now, we can improve later but this PR already solves @maxpoulin64's (and probably others') issue.